### DISCUSSION TITLE
Is there any estimate on how large the variation of the download statistics?

### DISCUSSION THREAD

**Discussion by cowboycodr** [psincraian/pepy#412] [General]
I recently started a project which I thought had no popularity.

However, Pepy states it has thousands of downloads. 

Is this accurate, and if so, to what extent?

**Comment by hugovk** [comment]
The numbers include mirroring services which make a copy of PyPI. 

See the FAQ for pypistats, another stats website which excludes mirrors from most charts, for more info:

https://pypistats.org/faqs#why-are-the-cumulative-download-counts-different-from-the-sum-of
